### DISCUSSION TITLE
Images not seen

### DISCUSSION THREAD

**Discussion by SasanK-0p** [erebe/greenclip#160] [Q&A]
When ever i copy any files or images (even less than 500 kb) only the directory to that file is shown not the image in rofi. How to make such that the images can be directly seen in rofi. Also how to remove file size cap. In klipper (kde clipboard) you can see the file name and image but in greenclip i only see the directory or the file.

Please help me!

**Comment by MaxGyver83** [comment]
I don't know how to show images in rofi. But I have added image preview to dmenu. Have a look at this patch: [png_images (dmenu)](https://tools.suckless.org/dmenu/patches/png_images/)
